examples/CustomCollectionView/Sample/MosaicCollectionLayoutDelegate.m

```diff
@@ -0,0 +1,175 @@
+//
+//  MosaicCollectionLayoutDelegate.m
+//  Sample
+//
+//  Created by McCallum, Levi on 11/22/15.
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//  Modifications to this file made after 4/13/2017 are: Copyright (c) 2017-present,
```

Comment:
This new file uses many logics from `MosaicCollectionViewLayout.m`, hence this version of Copyright Notice.